src/fluentd/src/fluent-plugin-pgjson/lib/fluent/plugin/out_pgjson.rb

```diff
@@ -209,9 +243,11 @@ def write(chunk)
             kind = record["objectSnapshot"]["kind"]
             trigger = record['objectSnapshotTrigger']
             log.info "[pgjson] object type #{kind} triggered by #{trigger} found in chunk #{hex_id}"
-            if trigger == "framework_retried" && kind == "Framework"
+            if trigger == "OnFrameworkRetry" && kind == "Framework"
               insert_framework hex_id, time, record
-            elsif trigger == "pod_deleted" && kind == "Pod"
+            elsif trigger == "OnTaskRetry" && kind == "Task"
+              insert_task hex_id, time, record
+            elsif trigger == "OnPodDeletion" && kind == "Pod"
               insert_pod hex_id, time, record
```

Comment:
We can also add this?

            elsif trigger == "OnTaskDeletion" && kind == "Task"
              insert_task hex_id, time, record

Reply:
No, we shouldn't add the last task object. We should keep the same logic for framework history table and task history table.

Reply:
Done.